src/App.jsx

```diff
@@ -1,35 +1,25 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
 
-function App() {
-  const [count, setCount] = useState(0)
+import { Routes, Route } from 'react-router-dom';
+import RegistrationForm from './components/registration-form';
+import Login from './components/login';
+import WanderPalRoutes from './routes/WanderPalRoutes';
+
+
 
+
+function App() {
```

Comment:
- [ ] Your frontend project consumes the API which is running on port 3001:

![image](https://github.com/danielamoreno699/wanderpal-frontend/assets/57421823/f5f94e6a-dd94-4b27-aac8-e2ab91d7fa77)

However, in your backend instructions, you instruct your users to run the project like this:

```bash
rails server
```

![image](https://github.com/danielamoreno699/wanderpal-frontend/assets/57421823/70bdbb1d-c2fa-4a4e-8530-35ff7069ec6b)


We need to tell the server to run on port 3001.   Kindly, on your backend readme, update your instructions so it runs on that port like this:

```bash
rails server -p 3001
```

That way, your frontend app will consume the API running on port 3001 successfully. 👍